src/styles/index.scss

```diff
@@ -8,7 +8,7 @@ html {
 
 body {
     font-size: rem($component-base-font-size);
-    font-family: sans-serif;
```

Comment:
Use a variable to define font family is a good approach or just leave them inline is better?

Reply:
I think it's fine to leave it inline for now, as it's only used once here. XD